src/setupTests.js

```diff
@@ -2,3 +2,9 @@ import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 configure({ adapter: new Adapter() });
+global.cloudinary = {
+  createUploadWidget: (response) => {
```

Comment:
May be export missing? I too am not sure. Just thinking if variable still flagged as unused could be the cause. But then if so, do we really need this?. Again I must say, I am not sure

Reply:
It can be accessed anywhere, remember this is configuration file